netbox/core/events.py

```diff
@@ -0,0 +1,33 @@
+from django.utils.translation import gettext as _
```

Comment:
This is only ever used in extras, should it be in extras?

Reply:
I haven't submitted any FRs yet, but I want to 1) allow plugins to register custom event types and 2) extend EventRule to allow mapping to any event type(s). So I opted to put this in `core` in anticipation of that work.